<launch>
  <include file="$(find astra_camera)/launch/stereo_s_u3.launch"/>

  <!-- Depth Image to Laser Scan Node -->
  <node name="depth_image_to_laser_scan" pkg="computervision_ex" type="depth_image_to_laser_scan" args="">

    <param name="scan_height"     type="int"    value="1"/> <!-- default: 1 pixel. Number of pixel rows used to generate laser scan. -->
    <param name="scan_time"       type="double" value="0.033"/> <!-- default:0.033, 30 FPS . Time between scans. -->
    <param name="range_min"       type="double" value="0.3"/> <!--default:0.45m. Ranges less than this are considered -Inf. -->
    <param name="range_max"       type="double" value="1.0"/> <!--default: 10m. Ranges less than this are considered +Inf. -->
    <param name="output_frame_id" type="str"    value="camera_depth_frame"/> <!--default: camera_depth_frame. Frame id of the laser scan. -->
    <param name="scan_h"          type="int"    value="100"/>
  </node>
	
</launch>
